presentation-api/controlling-ua/defaultRequest_success-manual.html

```diff
@@ -2,12 +2,13 @@
 <meta charset="utf-8">
 <title>Presentation API, testing to start a new presentation with an default request setup. (success - manual)</title>
 <link rel="author" title="Marius Wessel" href="http://www.fokus.fraunhofer.de">
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs">
 <link rel="help" href="http://w3c.github.io/presentation-api/#dfn-controlling-user-agent">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 
-<p>Click the button below and select the available casting device, to start the manual test.</p>
-<button onclick="startPresentation()">Start Presentation Test</button>
+<p>Click the button or the menu item for presentation on your browser (for example, "Cast"), to start the manual test.</p>
+<p>If your browser does not support <code>defaultRequest</code>, this test will not begin.</p>
```

Comment:
The test outcome should get reported to the test runner no matter what. There is no way to represent an optional test for the time being, I think. Last time I checked, the preferred outcome was to have the test FAIL. We could probably also prefix the test's label with `[Optional]` so that it's easy to spot that the failure may simply indicate lack of support for the feature.

To have the test fail, what should the user do? I suspect that if she just closes the window, she will have to state whether the test succeeded or failed to the test runner. It may be better to add a button that makes the test fail and instruct the user to press that button if the browser does not support `defaultRequest`.

That's open for discussion. I'm not saying this is the right approach. It's certainly not perfect :)